__tests__/Auth0Client.test.ts

```diff
@@ -1449,4 +1440,148 @@ describe('Auth0Client', () => {
       expect(decodedToken).toBeUndefined();
     });
   });
+
+  describe('getIdTokenClaims', () => {
+    describe('when loginWithRedirect', () => {
+      it('returns undefined if there is no cache', async () => {
+        const auth0 = setup();
+
+        jest.spyOn(auth0['cache'], 'get').mockReturnValueOnce(undefined);
+
+        const decodedToken = await auth0.getIdTokenClaims();
+        expect(decodedToken).toBeUndefined();
+      });
+
+      it('returns the ID token claims', async () => {
+        const auth0 = setup({ scope: 'foo' });
+        await loginWithRedirect(auth0);
+
+        expect(await auth0.getIdTokenClaims()).toHaveProperty('exp');
+        expect(await auth0.getIdTokenClaims()).not.toHaveProperty('me');
```

Comment:
As the IdTokenClaim uses `{ exp: Date.now() / 1000 + 86400 }` it is harder to verify the object's property being equal to an actual value. For that reason I used a `toHaveProperty` and a `not.toHaveProperty`. If you want I can mock Date.now() for this tests only, but I was not sure if that is needed.

Reply:
I think for the purposes of this test it's fine to check the existence of the property. We have other tests (JWT validation) that already check actual values.